From: sohee_yoon <sohee_yoon@tmax.co.kr>
Date: Mon, 18 Oct 2021 16:20:06 +0900
Subject: [feat] desktop-icons-ng : Hide external drives in the desktop

- Hide external drives in the desktop
- Change 'show-volumes' option value to false

[IMS] 272302
---
 extensions/desktop-icons-ng/schemas/10_desktop-icons-ng.gschema.override | 1 +
 1 file changed, 1 insertion(+)

diff --git a/extensions/desktop-icons-ng/schemas/10_desktop-icons-ng.gschema.override b/extensions/desktop-icons-ng/schemas/10_desktop-icons-ng.gschema.override
index 719bcba..cc7d799 100644
--- a/extensions/desktop-icons-ng/schemas/10_desktop-icons-ng.gschema.override
+++ b/extensions/desktop-icons-ng/schemas/10_desktop-icons-ng.gschema.override
@@ -1,3 +1,4 @@
 #Desktop Icons NG Drop Place
 [org.gnome.shell.extensions.ding]
 show-drop-place=false
+show-volumes=false
